Description: Generate the component descriptor with plexus-component-metadata instead of plexus-maven-plugin
Origin: backport, https://github.com/codehaus-plexus/plexus-pom/commit/3c5f784f
--- a/pom.xml
+++ b/pom.xml
@@ -105,6 +105,18 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-component-metadata</artifactId>
+        <executions>
+          <execution>
+            <id>process-classes</id>
+            <goals>
+              <goal>generate-metadata</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
     </plugins>
   </build>
 </project>
